cardpresent/src/main/java/com/fattmerchant/omni/Omni.kt

```diff
@@ -100,16 +100,21 @@ open class Omni internal constructor(internal var omniApi: OmniApi) {
                 mobileReaderDetails.anywhereCommerce?.let {
                     mutatedArgs["awc"] = it
                 }
+
+                // In case of Missing reader credentials
+                if (mutatedArgs["awc"] == null && mutatedArgs["nmi"] == null) {
+                    initialized = true
+                    error(OmniException("Could not get reader settings", "Your account does not have mobile reader credentials"))
+                    return@launch
+                }
             }
 
+            initialized = true
             InitializeDrivers(
                 mobileReaderDriverRepository,
                 mutatedArgs,
                 coroutineContext
             ).start(error)
```

Comment:
@omar-salah-elboredy We don't need to run this if there are no mobile reader credentials. If the credentials are null or blank, we can assume that the user does not care to have drivers initialized